torch/_decomp/decompositions.py

```diff
@@ -1543,9 +1543,9 @@ def native_group_norm_backward(
         lambda: f"Expect gamma to have {C} elements but got {gamma.numel() if gamma is not None else -1}",
     )
 
-    cpg, _rem = divmod(C, group)
```

Comment:
@bobrenjc93 @laithsakka might be better to also have a torch.sym_divmod so you don't have to do a transformation like this